src/rgw/rgw_admin.cc

```diff
@@ -9507,6 +9529,13 @@ int main(int argc, const char **argv)
       return -ret;
     }
 
+    if (opt_status) {
+        // status is being set/modified. Update sync_bucket()
+        ret = sync_policy_ctx.sync_bucket(group, bucket_op);
```

Comment:
what affect does sync_bucket() have?  It sounds like it tries to do a sync in the radosgw-admin process.  Is that intended?